backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5780840_sys_gh26855_Remove_MHD_ShipmentSchedule_Window.sql

```diff
@@ -0,0 +1,27 @@
+-- Run mode: SWING_CLIENT
+
+-- Column: M_ShipmentSchedule.BestBeforeDate
+-- 2025-12-12T09:27:01.609Z
+UPDATE AD_Column SET IsSelectionColumn='N',Updated=TO_TIMESTAMP('2025-12-12 09:27:01.609000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_Column_ID=589435
+;
```

Comment:
If the column is no longer a filter, I think it will not be a filter when element is displayed in a customer window.